ui/store/index.js

```diff
@@ -87,6 +107,107 @@ const actions = {
         status: 'danger'
       }, { root: true })
     }
+  },
+
+  toggleSelectionAll ({ commit, getters, state }) {
+    getters.areAllAccountsSelected ? commit('SET_SELECTED_ACCOUNTS', []) : commit('SET_SELECTED_ACCOUNTS', [...state.accounts])
+  },
+
+  async enableAccounts ({ commit, dispatch, state, rootGetters }) {
```

Comment:
Please refactor the function body of this and `disableAccounts` into one generic function. :-) still good to have dedicated functions for `enableAccounts` and `disableAccounts`, but they should call one generic function doing the heavy lifting.